### ISSUE TITLE
How to scan file in pod 

### ISSUE THREAD

**Issue by yangjinlogic** [fabric8io/kubernetes-client#2940]
The code can upload file.
kubernetesClientContainer.getClient().pods().inNamespace(namespace)   // <- Namespace of Pod
                .withName(pod)                  // <- Name of Pod
                .file(podPath)        // <- Target location of copied file inside Pod
                .upload(fileToUpload.toPath());        // <- Path of local file
but how to scan file in pod

**Comment by manusa** [comment]
What do you mean by "scan file"?

**Comment by yangjinlogic** [comment]
@manusa   I can upload file to container,but I wan to see the file and file directory

**Comment by yangjinlogic** [comment]
I can upload file to container,but I wan to see the file and file directory inside pod

**Comment by yangjinlogic** [comment]
@rohanKanojia help me ,my friend

**Comment by rohanKanojia** [comment]
What do you mean by `I want to see the file and file directory inside pod`? Do you mean you want to run `cat` or `ls` inside pod? I think you might already know how to do this.

**Comment by yangjinlogic** [comment]
@rohanKanojia  yes,I want to see the file and file directory inside pod,but I want to through 
 this way ：  kubernetesClientContainer.getClient().pods().inNamespace(namespace)   // <- Namespace of Pod
                .withName(pod) ...，but not  “ docker exec -it containerId ”  and cat or ls
